mkdocs.yml

```diff
@@ -30,88 +29,89 @@ theme:
 
 nav:
   - Learn:
-    - Welcome: learn/welcome.md
-    - Mainnet: learn/mainnet.md
-    - Features: learn/features.md
-    - Use cases: learn/use-cases.md
-    - Glossary: learn/glossary.md
-    - Networks: learn/networks.md
-    - Blog: learn/blog.md
-    - Explore:
-      - Explorers: explore/explorers.md
-      - Projects: explore/projects.md
-      - Sample apps: explore/sample-apps.md
-    - Advanced:
-      - Tech overview: learn/advanced/overview.md
-      - Data availability: learn/advanced/data-availability.md
+      - Welcome: learn/welcome.md
```

Comment:
all these formatting changes make it hard to see what *actually* changed.  If you're going to do a pass to clean up the formatting, ideally it'd be nice to do that as a separate commit that just does the formatting so that it can be reviewed independent of any functionality changes.

Reply:
I did not intend to change any of this, I think VS Code just decided to mess things up. 

The long and short of it is: 
- CLI Quickstart moved to CLI section
- New quickstart moved to where the Old Quickstart was.